stdlib/tkinter/ttk.pyi

```diff
@@ -1098,27 +1096,39 @@ class Treeview(Widget, tkinter.XView, tkinter.YView):
         open: bool = ...,
         tags: str | list[str] | tuple[str, ...] = ...,
     ) -> None: ...
-    def move(self, item: str, parent: str, index: int) -> None: ...
+    def move(self, item: str | int, parent: str, index: int) -> None: ...
     reattach = move
-    def next(self, item: str) -> str: ...  # returning empty string means last item
-    def parent(self, item: str) -> str: ...
-    def prev(self, item: str) -> str: ...  # returning empty string means first item
-    def see(self, item: str) -> None: ...
+    def next(self, item: str | int) -> str: ...  # returning empty string means last item
+    def parent(self, item: str | int) -> str: ...
+    def prev(self, item: str | int) -> str: ...  # returning empty string means first item
+    def see(self, item: str | int) -> None: ...
     if sys.version_info >= (3, 8):
         def selection(self) -> tuple[str, ...]: ...
     else:
         def selection(self, selop: Incomplete | None = ..., items: Incomplete | None = None) -> tuple[str, ...]: ...
 
-    def selection_set(self, items: str | list[str] | tuple[str, ...]) -> None: ...
-    def selection_add(self, items: str | list[str] | tuple[str, ...]) -> None: ...
-    def selection_remove(self, items: str | list[str] | tuple[str, ...]) -> None: ...
-    def selection_toggle(self, items: str | list[str] | tuple[str, ...]) -> None: ...
     @overload
-    def set(self, item: str, column: None = None, value: None = None) -> dict[str, Any]: ...
+    def selection_set(self, __items: str | int | list[str] | tuple[str, ...] | list[int] | tuple[int, ...]) -> None: ...
```

Comment:
Might be slightly simpler if the first overloads only accepted lists and tuples. Like this, for example:

```suggestion
    def selection_set(self, __items: list[str] | tuple[str, ...] | list[int] | tuple[int, ...]) -> None: ...
```

If you pass one `int` or `str`, it will match the overload with `*items` anyway.